woocommerce/woocommerce-android
# Issue/7532 add utm params for card reader pages

Closes: #7532 


### Description

In [Project Salas](https://github.com/woocommerce/woocommerce-ios/projects/51), we want to be able to understand the effectiveness of different JITM campaigns, initially those which link out to WooCommerce.com card reader purchase pages.

Adding UTM parameters to our links will allow us to attribute sales on WooCommerce.com, to the button in the apps the merchant tapped to get there.

This PR adds UTM parameters to the card reader purchase links.

### Testing instructions


Tap the CTA on the Upsell Card Reader banner (on the order list and payment methods screens) and the `Order card reader` row on the Payments menu.

Check that the following query parameters are added to the URL which is opened. 
Check that they are added to the `woocommercecom_product_view` tracks event.

- `utm_medium` – `woo_android`
- `utm_campaign` – `feature_announcement_card`, `payments_menu_item`: identifies whether this is a Feature Announcement Card, or a settings item.
- `utm_source` – `orders_list`, `payment_method`, or `payments_menu`
- `utm_term` – {STORE_ID}
- `utm_content` – `upsell_card_readers`: identifies the specific Feature Announcement Card which the tap came from. For other display locations, it's not required
